src/Tizen.NUI/src/public/NUIConstants.cs

```diff
@@ -1056,6 +1056,26 @@ public enum GlyphType
         VectorGlyph
     }
 
+    /// <summary>
+    /// Defines how a color is blended.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
```

Comment:
could you remove <since_tizen> tag ? and could you replace tizen_6.0 with tizen_next?
thank you.